### ISSUE TITLE
Link organisers' names to twitter or linked in

### ISSUE THREAD

**Issue by nolamesa** [euruko2013/site#61]
We should link everyone's name to their twitter or github or something

**Comment by fotos** [comment]
Yes, I am waiting on @sincospi to create a PR with everybody who has helped so far, to work on this one. This is also related to #48.
